pluribus/games/short_deck/state.py

```diff
@@ -77,14 +77,17 @@ def __init__(
             "river": 3,
             "show_down": 4,
         }
-        self._player_i_lut: Dict[str, List[int]] = {
-            betting_stage: [p_i for p_i in range(n_players)]
-            for betting_stage in self._betting_stage_to_round.keys()
-        }
-        pre_flop = self._player_i_lut["pre_flop"]
         # Rotate the big and small blind to the final positions for the pre
         # flop round only.
-        self._player_i_lut["pre_flop"] = pre_flop[-2:] + pre_flop[:-2]
+        player_i_order: List[int] = [p_i for p_i in range(n_players)]
+        self._player_i_lut: Dict[str, List[int]] = {
+            "pre_flop": player_i_order[-2:] + player_i_order[:-2],
```

Comment:
Actually, this reads like it would be incorrect, but maybe my brain is just not working (haven't slept in a minute). If post-flop the order is `[0,1,2]`, then that would mean that "Small Blind" is 0, which is fine (that would be `player_i_order`), but then before the flop the first player to act should be player 2, which would be the "Under the Gun" player. So this should be `player_i_order[2:] + player_i_order[:2]` I think?